File: 268-MissingNumber/268-MissingNumber.py
# Last updated: 8/13/2025, 8:20:24 PM
class Solution:
    def missingNumber(self, nums: List[int]) -> int:
        sum =  len(nums)
        for i in range (len(nums)) : 
            sum += (i - nums[i])
        return sum

File: 268-MissingNumber/268-MissingNumber.js
// Last updated: 8/13/2025, 8:20:30 PM
/**
 * @param {number[]} nums
 * @return {number}
 */
var missingNumber = function(nums) {
    let res = nums.length;
    for(let i=0;i<nums.length;i++){
        res += i - nums[i];
    }
    return res;
};